scripts/Create_table_appointments.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  date_on DATE,
+  time_on TIME,
+  comments TEXT,
+);
```

Comment:
We need to add tables for the doctor and patient's id's that reference the other tables.